frontend/src/scenes/hog-functions/configuration/hogFunctionConfigurationLogic.tsx

```diff
@@ -426,7 +426,13 @@ export const hogFunctionConfigurationLogic = kea<hogFunctionConfigurationLogicTy
                         }
                     }
 
-                    const res = await api.hogFunctions.getTemplate(props.templateId)
+                    let res: HogFunctionTemplateType | null = null
+                    if (props.templateId === 'native-dev-center') {
```

Comment:
Generally I like the idea of this PR alot. The only thing I'm less keen on is this bit. I'd much rather that we have a dedicated page for this, especially given there are _so_ many things that don't work (like saving etc.)

I think this could be "fine" for now, but I'd generally like to avoid muddying the template library with this magic template. Its a purely frontend thing so lets just do it purely on the frontend

Reply:
> I think this could be "fine" for now, but I'd generally like to avoid muddying the template library with this magic template. Its a purely frontend thing so lets just do it purely on the frontend

Removed the dummy template